### ISSUE TITLE
Question about subfolders

### ISSUE THREAD

**Issue by parvezalisha** [johvin/eslint-import-resolver-alias#7]
Hi,
I have a folder structure like:
```
src/helpers/
src/abcd/helpers/
```
If i have an alias "helpers" for src/helpers then the resolver tries to resolve src/abcs/helpers to that alias too and then eslint throws import/no-unresolved. 

Wanted to know if i am doing something wrong or if this is not supported.

Thanks

**Comment by parvezalisha** [comment]
My issue is resolved by changing the order of the paths. If i resolve src/abcd/helpers/ first then src/helpers/ doesn't throw errors. Thanks

**Comment by parvezalisha** [comment]
Can [this be mentioned|https://github.com/johvin/eslint-import-resolver-alias/issues/7#issuecomment-440775204] in the usage documentation?

**Comment by johvin** [comment]
@parvezalisha  Actually, I still don't know quite well what your configuration of alias resolver is, can you post it here so that I can be sure that description of the thrid note item in the usage section doesn't cover this situation. thanks.

**Comment by parvezalisha** [comment]
@johvin I have a folder structure like: 
```
src/helpers/
src/abcd/helpers/
```
and if i try to map it as below:
```
 map: [
          ['helpers', path.resolve(__dirname, './src/helpers')],
          ['abcd', path.resolve(__dirname, './src/abcd/helpers')], 
      ],
```
eslint throws unresolved path errors because the plugin tries to resolve src/abcd/helpers to the first helpers path. But if i change the order to:
```
 map: [
          ['abcd', path.resolve(__dirname, './src/abcd/helpers')], 
          ['helpers', path.resolve(__dirname, './src/helpers')],
      ],
```
everything works fine because now src/abcd/helpers gets resolved first and does not conflict with the other 'helpers' mapping.

**Comment by johvin** [comment]
@parvezalisha  Actually, description of the fourth note item in the usage section just explains the importance of the config rules' order, but maybe not clear enough for your situation. I will improve the notes with this issue soon. Many thanks.

**Comment by johvin** [comment]
@parvezalisha After much thought, I think your problem is caused by a bug of this plugin, not the order of configuration. The bug, actually, is specially designed as a feature to resolve `path/to/file` to `path/from/file` with the configuration of `['to', 'from']`, but it now seems problematic. I have fixed it in version `1.1.2` and welcome to upgrade to that. Sorry for the trouble to you and thanks.

**Comment by parvezalisha** [comment]
@johvin it works fine now. Thanks.
